index.bs

```diff
@@ -1900,7 +1859,10 @@ engine.
     :: The TPMT_PUBLIC structure (see [[TPMv2-Part2]] section 12.2.4) used by the TPM to represent the credential public key.
 
 : Signing procedure
-:: Concatenate the given <a>authenticatorData</a> and <a>clientDataHash</a> to form |attToBeSigned|.
+:: Let |authenticatorData| denote the [=authenticator data for the attestation=], and let |clientDataHash| denote the
```

Comment:
Yes, that is something I have wanted to do for a while as well. We should define what a signing procedure is in [Attestation Statement Formats](https://w3c.github.io/webauthn/#attestation-formats) and update the various formats to match. I started doing this, then realized that I'd also have to define data types and encoding for each argument, and so put it aside for a bit. Similar arguments can be made about the authenticator methods as well.